lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/data/ashrae_90_1_2007.spc_typ.json

```diff
@@ -13291,10 +13291,10 @@
       "occupancy_schedule": "OfficeSmall BLDG_OCC_SCH",
       "occupancy_activity_schedule": "OfficeSmall ACTIVITY_SCH",
       "is_residential": null,
-      "ventilation_standard": "ASHRAE 62.1-1999",
+      "ventilation_standard": "ASHRAE 62.1-2004",
       "ventilation_primary_space_type": "Office Buildings",
       "ventilation_secondary_space_type": "Whole Building",
-      "ventilation_per_area": 0.1,
+      "ventilation_per_area": 0.085,
```

Comment:
The ventilation standard was revised and the ventilation rates were updated accordingly.